CMakeLists.txt

```diff
@@ -68,6 +68,10 @@ if(LIBIGL_BUILD_TESTS)
 	add_subdirectory(tests)
 endif()
 
+# OSS-fuzz build
+# This is only used by OSS-fuzz
+option(OSS_FUZZ "Build for OSS-fuzz" OFF)
```

Comment:
Any new CMake/macro introduced should be prefixed by `LIBIGL_`. As it stands, I am not sure this one is needed at all (see below).